dropbox_uploader.sh

```diff
@@ -67,1768 +67,1785 @@ APP_CREATE_URL="https://www.dropbox.com/developers/apps"
 RESPONSE_FILE="$TMP_DIR/du_resp_$RANDOM"
 CHUNK_FILE="$TMP_DIR/du_chunk_$RANDOM"
 TEMP_FILE="$TMP_DIR/du_tmp_$RANDOM"
-AUTH_ACCESS_TOKEN_EXPIRE="0"
```

Comment:
This variable was here unused. So, this variable should belong to OAUTH_ACCESS_TOKEN_EXPIRE a global variable that need to be declared.